README.md

```diff
@@ -223,6 +223,29 @@ Let's use `babel` as an example.
 - `babel-generator` and `source-map` (among others) are dependencies of `babel-core`.
 -  `babel-core`'s [`package.json`](https://github.com/babel/babel/blob/13c961d29d76ccd38b1fc61333a874072e9a8d6a/packages/babel-core/package.json#L28-L47) lists both these packages as keys in `dependencies`, as shown below.
 
+### add
+
+```sh
+$ lerna add <package>[@version] [--dev]
+```
+
+Add `package` as dependency to packages in in the current Lerna repo.
+
+When run, this command will:
+
+1. Add `package` to each applicable package. Applicable are packages that are not `package` and are in scope
```

Comment:
(it occurs to me that the `--hoist` behaviour is already dealt with by calling bootstrap at the end of execute, _as long as_ we are also passing along `this.options` instead of the flags)

Reply:
Addressed installing external dependencies via https://github.com/lerna/lerna/pull/1069/commits/4be0d91870596bf91f6e3ad58b178e66213806be. 

The change got bigger than I'd like due to the fact I switched to an async control flow. 

This is needed due to the network requests we send to determine the correct remote package version ranges to persist.

Reply:
This type of thing could be easier to build with `async - await`. Is there a reason is not used in the codebase so far?